docs/index.mdx

```diff
@@ -3,26 +3,32 @@ title: Main
 description: The powerful Flutter-first UI testing framework. Ready for action!
 ---
 
-# About
+# About Patrol framework
 
 ![Patrol promotial graphics][promo_graphics]
 
 Patrol is a new testing framework for Flutter. It builds on top of Flutter's
 existing test tooling to let you do things which were previously impossible.
 
-Patrol lets you [access native features of the platform][native] that the
+Patrol is combined of 3 packages - one dedicated to widget testing, and other
+two made for UI testing. Together they provide enhanced testing experience
+both for developers and for automating testers.
+
+### patrol
+`patrol` package lets you [access native features of the platform][native] that the
```

Comment:
Missing new line between header and the paragraph, you guys use some formatter for mdx files?

Reply:
in my case the formatter don't work properly with mdx, also didn't find any dedicated to mdx, only md. If you have something on your mind, I'm happy to use it